models/demos/wormhole/llama31_8b/demo/demo_trace.py

```diff
@@ -585,7 +585,7 @@ def run_llama_demo(user_input, batch_size, device, instruct_mode, is_ci_env, num
         "instruct_weights-3_batch",
     ],
 )
-@pytest.mark.parametrize("device_params", [{"trace_region_size": 7860224, "num_command_queues": 2}], indirect=True)
```

Comment:
is this related to this change?

Reply:
Yeah. there was a failure in the pipeline when I ran this branch over the CI.

Reply:
ok, any idea how this change affects this?

Reply:
Yeah i ran the pipeline again after this change and it passed